integration-tests/integration/prisonerProfile/prisonerQuickLook.spec.js

```diff
@@ -9,7 +9,7 @@ const quickLookFullDetails = {
   prisonerDetails: [
     {
       imprisonmentStatusDesc: 'Adult Imprisonment Without Option CJA03',
-      dateOfBirth: '1998-12-01',
+      dateOfBirth: '1999-12-01',
```

Comment:
Be better to set this date dynamically using moment.

Reply:
will be done in separate PR